Frameworks/OCMock.framework/Modules

```diff
@@ -0,0 +1 @@
+Versions/Current/Modules
```

Comment:
Could we not get this framework through the swift package manager? I worry about static dependencies as they have a way of getting out of date

Reply:
Nope, doesn't support ... but it's only for tests so the reason I updated it is the new M1 support, and we won't need it for years again 😂